pkg/apis/forklift/v1beta1/plan/vm.go

```diff
@@ -64,6 +64,8 @@ type VMStatus struct {
 	RestorePowerState VMPowerState `json:"restorePowerState,omitempty"`
 	// The firmware type detected from the OVF file produced by virt-v2v.
 	Firmware string `json:"firmware,omitempty"`
+	// The Operation System detected by virt-v2v.
```

Comment:
```suggestion
	// The Operating System detected by virt-v2v.
```
and please re-generate the yaml files